kde-frameworks/kapidox/kapidox-5.21.0.ebuild

```diff
@@ -4,16 +4,16 @@
 
 EAPI=6
 
-KDE_DEBUG="false"
-KDE_DOXYGEN="false"
-KDE_TEST="false"
 DISTUTILS_SINGLE_IMPL="true"
 PYTHON_COMPAT=( python2_7 )
-inherit kde5 distutils-r1
```

Comment:
possible and i did that at first, but that would still keep the cmake dependency. not sure what purpose kde5.eclass has when it takes more lines to disable stuff than it saves in providing metadata. ;)